frontend/src/App.tsx

```diff
@@ -36,6 +37,17 @@ const App = () => {
           <>
             <AppRouter />
             <OnLoadActions />
+            <ToastContainer
```

Comment:
This is the root toast container in the DOM. All toasts are displayed within this container. Also, global toast defaults are defined here. These defaults can be overriden when creating a toast as well via:

toast('message' { toast_specific_options})

Reply:
I think having a consistent location for all toasts makes sense.